src/components/TradeWidget/index.tsx

```diff
@@ -64,17 +64,22 @@ const TradeWidget: React.FC = () => {
   const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId), [fallBackNetworkId])
 
   // Listen on manual changes to URL search query
-  const { sell: sellTokenSymbol, receive: receiveTokenSymbol } = useParams()
+  const { sell: sellTokenSymbol, sellAmt, receive: receiveTokenSymbol, receiveAmt } = useParams()
```

Comment:
I wouldn't shorten the name. `sellAmount`

Also, although we could change `sell` for `sellSymbol` too, maybe is best to leave it. It seams reasonable to make `sell` to accept in the future also addresses. So you can virtually encode any trade of ANY token pair in the url.

Reply:
Updated